wordpress-mobile/WordPress-Android
# Lightmode Avatar Placeholder Issue

Fixes #13918

This fixes the issue of the placeholder image not being shown in light mode. The placeholder image had a white colour which in white background(light mode) would not be visible. The placeholder and fallback icon in case of error were replaced with another icon which is used as the placeholder for an avatar. To be more precise, the placeholder resource `ImageType.USER` is replaced with `ImageTypeAVATAR_WITH_BACKGROUND`. The `ImageType.USER` is unused and so is deleted. 


### Before 

| Light Mode | Dark Mode |
| :---        |    :----:   |  
|![Screenshot_20220124-152007](https://user-images.githubusercontent.com/17463767/150773910-9fa22d1e-f12c-4ba0-a613-8930c7b013dc.jpg)|![Screenshot_20220124-151955](https://user-images.githubusercontent.com/17463767/150773898-bcbaddbf-3cdd-4fb3-aa14-1682a2186639.jpg)|


### After the fix 

| Light Mode | Dark Mode |
| :---        |    :----:   |  
| ![Screenshot_20220124-151906](https://user-images.githubusercontent.com/17463767/150773378-ccc60955-129b-4126-afde-8fd48a0267cd.jpg) | ![Screenshot_20220124-151915](https://user-images.githubusercontent.com/17463767/150773756-cf7109bd-7ba0-4ee7-9f74-c34afe9c37b5.jpg) |






To test:

1. Launch the app in light mode
2. Login with a Wpcom or use a self-hosted account that doesn't have an account logo.
3. Go to My Site screen.
4. Notice that the user placeholder is displayed 

## Regression Notes
1. Potential unintended areas of impact 
    1. Places where the placeholder image type is `ImageType.USER` will crash
    2. Placeholder not being displayed properly 

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual tests 

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
